### PR TITLE
adding engine election aurora

### PR DESCRIPTION
**Pull request by venturel**
Adding option to choose the engine in Aurora.
Also done bit of formatting

### CODE REVIEW

File: aurora/main.tf [hunk 1]
```diff
@@ -9,6 +9,7 @@ resource "aws_db_subnet_group" "aurora" {
 }
 
 resource "aws_db_parameter_group" "aurora_mysql" {
+  count       = "${length(var.instance_parameter_group_name) == 0 ? 1 : 0}"
```
Comment (SamClinckspoor) [inline aurora/main.tf:12]: why is this change needed for the engine version?

**Review by iuriaranda — APPROVED** [review]

File: aurora/main.tf (hunk 1 above)
Comment (venturel) [reply aurora/main.tf:12]: basically we don't create a mysql parameter group for mysql if another one is specified

**Review by SamClinckspoor — APPROVED** [review]
